### ISSUE TITLE
Error with send_to_nvimcom

### ISSUE THREAD

**Issue by PMassicotte** [R-nvim/cmp-r#1]
Since this morning I am getting this error as soon as I open parenthesis (i.e. when auto complete is kicking in).

![image](https://github.com/R-nvim/cmp-r/assets/4519221/943ae34f-eb87-424b-a3d7-c13e5e37755c)

Was working fine until today.

Any ideas?

**Comment by jalvesaq** [comment]
This was happening to me if I uninstalled tmp-R-Nvim, and I thought I had fixed it on https://github.com/R-nvim/cmp-r/commit/18cecfc58f5b5a56c20b1dbb83d4a13fe982a8f7

Maybe lazy.nvim is loading cmp-r before tmp-Nvim-R, when `require("r.run")` is not available yet. We have to find a solution that fails silently (or with a useful warning message) if tmp-Nvim-R is not installed, but never fails if it is installed. Perhaps we have to postpone the definition of `send_to_nvimcom` to the moment when it will be used for the first time.
